.ahub/sam/exclude.txt

```diff
@@ -0,0 +1,15 @@
+/ONE/compiler/ann-api/include/NeuralNetworks.h
+/ONE/compiler/ann-ref
+/ONE/compiler/nnc/backends/soft_backend/code_snippets/eigen.def
+/ONE/tests
+/ONE/runtime/onert/frontend/circle_schema/include/circle_schema_generated.h
+/ONE/runtime/onert/frontend/tflite/src/tflite_schema_generated.h
+/ONE/runtime/nnapi-header/include/NeuralNetworks.h
+/ONE/runtime/nnapi-header/include/NeuralNetworksExtensions.h
+/ONE/runtime/libs/nnapi
+/ONE/runtime/libs/profiling
+/ONE/runtime/libs/tflite/port
+/ONE/runtime/3rdparty
+/ONE/compute
+/ONE/runtime/contrib
+/ONE/externals
```

Comment:
Note
- These are merged exclude from previous configuration for compiler and runtime.
- I really hope to add comment for each line, but not sure it is possible. Will do that in separate PR if possible.